ansible/roles_ocp_workloads/ocp4_workload_serverless/tasks/remove_workload.yml

```diff
@@ -8,6 +8,15 @@
     name: knative-serving
     namespace: knative-serving
 
+- name: Remove Knative Eventing Installation
+  when: ocp4_workload_serverless_install_eventing | bool
```

Comment:
Shouldn't this be `not`?

Reply:
It's the remove workload. So when this is true that means that eventing was installed.